### PR TITLE
feat(frontend): support show all (#3665)

### PR DESCRIPTION
**Pull request by chenzl25**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

As title.

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

### Types of user-facing changes

Please keep the types that apply to your changes, and remove those that do not apply.

* SQL commands, functions, and operators 
       support show config like SHOW ALL.

### Release note

Please create a release note for your changes. In the release note, focus on the impact on users, and mention the environment or conditions where the impact may occur.

### CODE REVIEW

**Review by yezizp2012 — APPROVED** [review]
LGTM

**Review by TennyZhuang — APPROVED** [review]

File: e2e_test/batch/basic/basic.slt.part [hunk 1]
```diff
@@ -35,3 +35,13 @@ query I
...
+
+query I
+show all;
+----
+rw_implicit_flush true If `RW_IMPLICIT_FLUSH` is on, then every INSERT/UPDATE/DELETE statement will block until the entire dataflow is refreshed.
+rw_force_delta_join false To force the usage of delta join in streaming execution.
+query_mode distributed A temporary config variable to force query running in either local or distributed mode.
+extra_float_digits 1 Sets the number of digits displayed for floating-point values.
+application_name (empty) Sets the application name to be reported in statistics and logs.
+date_style (empty) It is typically set by an application upon connection to the server.
```
[the remark this answers is not in the document]
Comment (chenzl25) [reply e2e_test/batch/basic/basic.slt.part:47]: It seems like sql logic test separate each column by space.
Actually, there are three columns here : name, setting and description.

[the remark this answers is not in the document]
Comment (BowenXiao1999) [reply e2e_test/batch/basic/basic.slt.part:47]: Got it

**Comment by chenzl25** [comment]
> You can just remove `show all` cases Or move `show` related cases into a new file and only run it in local or distributed mode.
OK, I just remove show all test. It is hard to make it deterministic.

**Pull request merged**